Project.toml

```
@@ -1,30 +1,31 @@
 name = "MixedModelsSim"
 uuid = "d5ae56c5-23ca-4a1f-b505-9fc4796fc1fe"
 authors = ["Phillip Alday", "Douglas Bates", "Lisa DeBruine", "Reinhold Kliegl"]
-version = "0.1.0"
+version = "0.2.0"
 
 [deps]
 DataFrames = "a93c6f00-e57d-5684-b7b6-d8193f3e46c0"
+Documenter = "e30172f5-a6a5-5a46-863b-614d45cd2de4"
 MixedModels = "ff71e718-51f3-5ec2-a782-8ffcbfa3c316"
 PooledArrays = "2dfb63ee-cc39-5dd5-95bd-886bf059d720"
+ProgressMeter = "92933f4c-e287-5a05-a399-4b506db050ca"
 Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
 StaticArrays = "90137ffa-7385-5640-81b9-e52037218182"
 Statistics = "10745b16-79ce-11e8-11f9-7d13ad32a3b2"
 Tables = "bd369af6-aec1-5ad0-b16a-f7cc5008161c"
 
 [compat]
-DataFrames = "0.20"
-Documenter = "0.24"
-MixedModels = "2"
+DataFrames = "0.21"
+Documenter = "0.23, 0.24, 0.25"
+MixedModels = "2, 3"
 PooledArrays = "0.5"
 StaticArrays = "0.12"
-Tables = "0.2"
-julia = "1.3"
+Tables = "1.0"
```

Even though I am the one who changed this to its current version, I wonder if this should just be
```suggestion
Tables = "1"
```

---

Good question. If I recall correctly, CompatHelper really wants you to have `1.0` (and indeed that's what it's suggested in #18). I think there is some fine print on when this makes a difference, but I would just leave it for now.